llvm/test/CodeGen/AMDGPU/fix-sgpr-copies-f16-true16.mir

```diff
@@ -1,6 +1,7 @@
 # NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 3
 # RUN: llc -mtriple=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -run-pass=si-fix-sgpr-copies -verify-machineinstrs -o - %s | FileCheck --check-prefixes=GCN %s
-#
+# XFAIL: *
+# FIXME. reenable after fix-sgpr-copies is updated for true16 flow 
```

Comment:
Might be useful to mark it with FIXME-TRUE16 here upstream as well?

Reply:
done